x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/index.test.ts

```diff
@@ -41,9 +41,18 @@ describe('EndpointList store concerns', () => {
         total: 0,
         loading: false,
         error: undefined,
-        details: undefined,
-        detailsLoading: false,
-        detailsError: undefined,
+        endpointDetails: {
+          activityLog: {
```

Comment:
ℹ️  `AsyncResourceState` can simplify the management of these states

Reply:
done c5ade7bd293be1f5ef9e19873c68513f16f399cf